AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -692,32 +704,29 @@ - (void)startDelayedCrashProcessing {
   // This must be performed asynchronously to prevent a deadlock with 'unprocessedCrashReports'.
   dispatch_time_t delay = dispatch_time(DISPATCH_TIME_NOW, kMSCrashProcessingDelay);
   dispatch_after(delay, dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
-    [self startCrashProcessing];
```

Comment:
Why do you touch this method? I think you just add an observer and call a method to process crashes when it is back to foreground.

Reply:
Background check should move outside (here) to play well with "dispatch_once" logic

Reply:
I'm curious, what's the scenarios that it is covering? Background -> Foreground -> Background -> Foreground?

Reply:
Just "Background -> Start, try to process -> Foreground -> process"